src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/__init__.py

```diff
@@ -19,8 +19,12 @@ def handler(ctx, **kwargs):
             if not ran_before:
                 extension_text = """
 Comprehensive IoT data-plane functionality is available
-in the Azure IoT CLI Extension. For more info and install guide
-go to https://github.com/Azure/azure-iot-cli-extension
+in the Azure IoT CLI Extension.
```

Comment:
Could this line possible be on the previous line? Were you able to test this change and see how it looks?

Reply:
It definitely could. Just wanted to insert the install command in between the two sentences, with as little change to the existing message / formatting as possible, and the original looked like it was trying to stay under a certain character limit per line. I'll move it up.